docker-compose.yaml

```diff
@@ -1,39 +1,39 @@
-version: "3.8"
- 
 services:
   backend:
     build: .
-    container_name: flask_backend
+    container_name: cesaxis_backend
     ports:
       - "5001:5000"
     environment:
       - FLASK_ENV=development
-      - MYSQL_HOST=db
-      - MYSQL_USER=user
-      - MYSQL_PASSWORD=password
-      - MYSQL_DATABASE=mydatabase
+      - FLASK_DEBUG=1
+      - MYSQL_HOST=${MYSQL_HOST}
+      - MYSQL_USER=${MYSQL_USER}
+      - MYSQL_PASSWORD=${MYSQL_PASSWORD}
+      - MYSQL_DATABASE=${MYSQL_DATABASE}
```

Comment:
@noelia1804 and @davidfnatixis, better we define the env vars like this so that it uses the ones coming from the `.env` file (after the dev has copied the `.env.example`)